usaspending_api/disaster/v2/views/agency/loans.py

```diff
@@ -44,25 +43,49 @@ def route_agency_loans_backend(request, *args, **kwargs):
     return route_agency_loans_backend
 
 
-class LoansByAgencyViewSet(LoansPaginationMixin, LoansMixin, FabaOutlayMixin, DisasterBase):
+class LoansByAgencyViewSet(
+    LoansPaginationMixin, ElasticsearchAccountDisasterBase, LoansMixin, FabaOutlayMixin, DisasterBase
```

Comment:
I think it was because this agency endpoint has a dual impl. If `award_type_codes` provided - hit awards data using subtier agency. If not hit postgres data to get top-tier agency sum. 

But, now that toptier is done in ES, is `DiasterBase` still needed? May it is if this impl handles when "Total" awards (file b data) is also selected?

Reply:
There were other ways that dual-implementation was handled. Creating the view class from two "base" classes really muddies the water